README.md

```diff
@@ -1,2 +1,11 @@
 # yokedox
 Yoked ox
+
+## To build Yokedox locally
+
+#. Cd into your cloned Yokedox repository, and then `/cli`
+#. Run `npm install` to install dependencies
+#. Run `npm run generate` to...?
```

Comment:
This is not a usual required step. This generates the Typescript types from the doclet JSON schema. (See plugins/JsonDocletJava8/doclet8.schema.json and cli/src/plugins/javadoc/doclet8.d.ts ). One only needs to do this if the doclet schema has changed.

Reply:
Hmm, in that case, presuming someone must do this the first time they build locally?

Reply:
No, the generated result is committed.

Reply:
Alrighty, removed this. Thanks!